### PR TITLE
[XLIFF] Import - bypass element not found exceptions

### LINKED ISSUE #5764: XLIFF Import throws Error (objects)
### Expected behavior
XLIFF Import (for objects) works properly, although one or more objects have been deleted in the meantime after the export.

### Actual behavior
XLIFF Import (for objects) crashes at the object, which doesn't exist anymore and doesn't go on with the following objects

### Steps to reproduce
- Make a XLIFF Export of a folder with objects in it.
- Delete an object in this folder.
- Try to import translated XLIFF file --> throws Error

Return Line:
https://github.com/pimcore/pimcore/blob/v6.4.2/bundles/AdminBundle/Controller/Admin/TranslationController.php#L901

### PR DESCRIPTION
**Pull request by dvesh3**
## Changes in this pull request  
Resolves #5764

### CODE REVIEW

File: bundles/AdminBundle/Controller/Admin/TranslationController.php [hunk 1]
```diff
@@ -899,9 +899,15 @@ public function xliffImportElementAction(Request $request, ImportDataExtractorIn
             $importerService->import($attributeSet);
         } catch (\Exception $e) {
             Logger::err($e->getMessage());
+            $success = false;
+
```
Comment (weisswurstkanone) [inline bundles/AdminBundle/Controller/Admin/TranslationController.php]: I guess this can be simplified

Comment (dvesh3) [reply bundles/AdminBundle/Controller/Admin/TranslationController.php]: @weisswurstkanone how?

File: bundles/AdminBundle/Controller/Admin/TranslationController.php [hunk 2]
```diff
@@ -899,9 +899,11 @@ public function xliffImportElementAction(Request $request, ImportDataExtractorIn
             $importerService->import($attributeSet);
         } catch (\Exception $e) {
             Logger::err($e->getMessage());
+            $success = $e instanceof Element\Exception\ElementNotFoundException ?? false;
```
Comment (jdreesen) [inline bundles/AdminBundle/Controller/Admin/TranslationController.php]: I doubt that the result of `instanceof` can ever be `null`.

Comment (dvesh3) [reply bundles/AdminBundle/Controller/Admin/TranslationController.php]: true!!....changed it 😉 thanks!

**Comment by brusch** [comment]
@dvesh3 just pushed an approach that required less code, what do you think?

**Comment by dvesh3** [comment]
@brusch I thought about the same approach initially but was just concerned that it will silently ignore the elements not found during the process. I believe user should be notified in some way(logs) that some jobs failed due to an exception.

**Comment by brusch** [comment]
Just added logging, but with a lower severity (`warn`), because actually it doesn't break the import process.

**Pull request merged**